ftpretty.py

```diff
@@ -31,18 +31,25 @@
 class ftpretty(object):
     """ A wrapper for FTP connections """
     conn = None
+    port = 21
```

Comment:
You should set port to None rather than 21, 21 will be incorrect for TLS connections. Then below only set port  if value is not None, ie. set via kwarg.